django/project/admin.py

```diff
@@ -228,9 +229,65 @@ def has_add_permission(self, request):
+        labels = {'investment_to_date': 'Cumulative investment.'}
+        widgets = {'innovation_hub': forms.CheckboxInput,
+                   'icon': forms.Select(choices=[(1, 'education'),
+                                                 (2, 'nutrition'),
+                                                 (3, 'health'),
+                                                 (4, 'child_protection'),
+                                                 (5, 'wash'),
+                                                 (6, 'aids'),
+                                                 (7, 'social_inclusion'),
+                                                 (8, 'food_health'),
+                                                 (9, 'affected_population'),
+                                                 (10, 'children'),
+                                                 (11, 'gender_balance'),
+                                                 (14, 'infant'),
+                                                 (15, 'breast_feeding'),
+                                                 (16, 'children_with_disabilities'),
+                                                 (27, 'pregnant'),
+                                                 (29, 'mental_health'),
+                                                 (30, 'mother_and_baby'),
+                                                 (34, '5year_old_girl'),
+                                                 (38, 'conflict'),
+                                                 (42, 'tsunami'),
+                                                 (45, 'epidemic'),
+                                                 (52, 'tent'),
+                                                 (69, 'medical_supplies'),
+                                                 (70, 'vaccines'),
+                                                 (71, 'psychosocial_support'),
+                                                 (72, 'headquarters'),
+                                                 (73, 'regional_office'),
+                                                 (92, 'partnership'),
+                                                 (97, 'advocacy'),
+                                                 (113, 'innovation'),
+                                                 (116, 'community_building'),
+                                                 (164, 'emergency')])}
+
+
 class PortfolioAdmin(AllObjectsAdmin):
-    list_display = ['__str__', 'description', 'status', 'created', 'icon', 'managers_list', 'is_active']
-    fields = ['name', 'description', 'status', 'managers', 'icon', 'is_active']
+    list_display = ['__str__', 'description', 'status',
+                    'created', 'icon', 'managers_list', 'is_active', 'innovation_hub', 'investment_to_date']
+    inlines = [ProblemStatementsInline]
+    form = PortfolioForm
+
+    def formfield_for_manytomany(self, db_field, request, **kwargs):  # pragma: no cover
```

Comment:
this is not needed, please use `filter_horizontal` for the field

Reply:
yes, that is a possible way to do it
however it did not worked in any default setup, this is a workaround for this exception
lets have a 5 min meeting about this, else i do not see any easier way to handle this now

Reply:
resolved ✅